docs/odl9-release-planning.md

```diff
@@ -0,0 +1,91 @@
+| **[2478](https://github.com/OData/odata.net/issues/2478): Remove `JSONP` support** | `JSONP` feature was marked as obsolete in ODL 8. Drop it altogether in ODL 9 |  Core |  | |
+| **[2530](https://github.com/OData/odata.net/issues/2530): Schema annotations not included in vocabulary annotations for model** | Edm model does not load in-line schema annotations and errors when out-of-line annotations are included |  Edm |  | |
+| **[2064](https://github.com/OData/odata.net/issues/2064): Explore improving perf around `ICollection<T>.Contains()`** | See [`ClientPropertyAnnotation`](https://github.com/OData/odata.net/blob/aba9a63ec5ca1cf8180ab83df8fe953f018e4785/src/Microsoft.OData.Client/Metadata/ClientPropertyAnnotation.cs#L172) class |  Core |  | |
+| **[2105](https://github.com/OData/odata.net/issues/2105): AddLink/SetLink to a navigation property in a complex type** | Support for navigation properties on complex types was added in [1743](https://github.com/OData/odata.net/pull/1743). However, there is currently no way to apply it to a complex type in OData client |  Client |  | |
+| **[1218](https://github.com/OData/odata.net/issues/1218): Remove explicit `HttpVersion` check in `ODataMultipartMixedBatchReader`** | See [`HttpVersionInBatching`](https://github.com/OData/odata.net/blob/aba9a63ec5ca1cf8180ab83df8fe953f018e4785/src/Microsoft.OData.Core/ODataConstants.cs#L73) constant |  Core |  | |
+| **[3181](https://github.com/OData/odata.net/issues/3181): Remove obsolete `ODataMessageReaderSettings.ReadUntypedAsString property`** |  |  Core |  | |
+| **[3182](https://github.com/OData/odata.net/issues/3182): Remove obsolete `DataServiceContext.Timeout` property** |  |  Client |  | |
+| **[3183](https://github.com/OData/odata.net/issues/3183): Resolve the `SYSLIB0051` obsoletion warning reported on `DataServiceClientException`** |  |  Client |  | |
+
+[//]: # "**[](https://github.com/OData/odata.net/issues/): ** |  |  Core |  | |"
+
+## Milestones
+### 1. Assessment Phase (February - March 2025)
+  - Review each proposed item to determine its relevance and priority for the ODL 9 release.
+  - Conduct feasibility analysis to assess the complexity and resources required for implementation.
+
+### 2. Development Phase (March - September 2025)
+  - Implement selected features and address prioritized issues.
+  - Perform internal testing to ensure stability and performance improvements.
+
+### 3. Preview Releases (March 2025)
+  - Release a preview version to gather feedback from the community.
+  - Address any critical issues identified during the preview phase.
+
+### 4. Beta Releases (August 2025)
+ - Release a beta version incorporating feedback from the preview phase.
+  - Continue to monitor and address any reported issues.
+
+### 5. Release Candidate (October 2025)

```

Comment:
When is the cutoff for new features/changes?

Reply:
@habbes 
> ### 2. Development Phase (March - September 2025)
  - Implement selected features and address prioritized issues.
  - Perform internal testing to ensure stability and performance improvements.

Cutoff would be end of September but use October to finalize as we write documentation where necessary
> ### 5. Release Candidate (October 2025)
  - Finalize features and perform thorough testing.
  - Prepare documentation and resources for the GA release.